test/mocks/MockZX1271Wallet.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+import "@openzeppelin/contracts/cryptography/ECDSA.sol";
+import "contracts/interfaces/IERC1271Wallet.sol";
+
+contract MockZX1271Wallet is IERC1271Wallet {
+    bytes4 public constant ZX1271_MAGICVALUE = bytes4(keccak256("isValidWalletSignature(bytes32,address,bytes)"));
+
+    address public operator;
+
+    constructor(address _operator) {
+        operator = _operator;
+    }
+
+    function isValidSignature(bytes calldata _data, bytes calldata _signature) external view override returns (bytes4 magicValue) {
+        require(operator == ECDSA.recover(keccak256(_data), _signature), "MockERC1271Wallet: invalid signature");
```

Comment:
Q: Does the production version ZX1271Wallet share the same behavior of revert when signature is not valid?

Reply:
It varies from different implementations. One of two ZX1271 wallets in our production reverts when invalid (https://etherscan.io/address/0x3b938e9525e14361091ee464d8acec291b3cae50#code). Another one didn't upload their code to etherscan.

Reply:
Added two regression to make sure it won't break anything 6fcccf2afdf7cac02c8aba2dbe174378bfe1c793